LCM/scripts/Register.py

```diff
@@ -52,7 +53,7 @@ def usage():
 if inArgument:
    Variables[currentArgument] = arg
 
-AcceptableOptions = ["registrationkey", "serverurl", "refreshfrequencymins", "configurationmodefrequencymins", "configurationmode", "refreshmode", "help"]
```

Comment:
I would recommend using [argparse](https://docs.python.org/2.7/library/argparse.html) instead of re-implementing it. Otherwise signed off